src/prefixer.js

```diff
@@ -47,7 +47,13 @@ function getPrefixer(userAgent: ?string): InlineStylePrefixer {
   }
 
   if (!_cachedPrefixer || actualUserAgent !== _lastUserAgent) {
-    _cachedPrefixer = new InlineStylePrefixer({userAgent: actualUserAgent});
+    if (actualUserAgent === 'all') {
+      _cachedPrefixer = {
+        prefix: InlineStylePrefixer.prefixAll,
```

Comment:
This will break flow (static typing), since it doesn't have all the properties of the actual [`InlineStylePrefixer`](https://github.com/FormidableLabs/radium/blob/master/interfaces/inline-style-prefixer.js).

We can definitely have a separate interface for `_cachedPrefixer`, and keep in mind that for this to be complete we also need to add all the prefixes for `prefixedKeyframes` below.